### PR TITLE
Add Karpenter doc

### PR DESCRIPTION
**Pull request by jul-dan**

### CODE REVIEW

File: website/docs/using-qovery/configuration/clusters.md [hunk 1]
```diff
@@ -210,10 +210,23 @@ To confirm, click `Next`.
...
+<p align="center">
+  <img src="/img/configuration/clusters/karpenter.png" alt="Enable Karpenter" />
+</p>
+
+Today, only new non-production clusters are supported. It means you won't be able to enable it on your already existing cluster. It will be supported soon.
+
+By activating Karpenter, you have to set:
+* `Disk size`: select the size of the disks to be attached to your cluster instances (to locally store container images etc..).
+* `Default node architecture`: your application will be build using this architecture by default.
+* `Spot instances`: In order to reduce even more your costs, you can also enable the spot instances on your clusters. Spot instances cost up to 90% less compared to On-Demand prices. But keep in mind that ppot instances can be terminated by the cloud provider at any time. Check this [documentation](https://aws.amazon.com/ec2/spot/) for more information.
```
Comment (pggb25) [inline website/docs/using-qovery/configuration/clusters.md:228]: typo

File: website/docs/using-qovery/configuration/clusters.md [hunk 2]
```diff
@@ -210,10 +210,23 @@ To confirm, click `Next`.
...
 
+You have the possibility to enable `Karpenter` autoscaler to improve the efficiency and cost of running workloads on your cluster. You can check the [official documentation](https://karpenter.sh/docs/) for more information.
+
+<p align="center">
+  <img src="/img/configuration/clusters/karpenter.png" alt="Enable Karpenter" />
+</p>
+
+Today, only new non-production clusters are supported. It means you won't be able to enable it on your already existing cluster. It will be supported soon.
+
+By activating Karpenter, you have to set:
```
Comment (pggb25) [inline website/docs/using-qovery/configuration/clusters.md:225]: should we inform that stateful set will not run on spot instance event if the flag is enabled?

File: website/docs/using-qovery/configuration/clusters.md [hunk 3]
```diff
@@ -210,10 +210,23 @@ To confirm, click `Next`.
...
-* `Instance type`: select the type of [worker nodes][docs.using-qovery.configuration.clusters#what-is-a-cluster] you want to deploy to your cluster:
 * `Disk size`: select the size of the disks to be attached to your cluster instances (to locally store container images etc..). Setting available only on AWS.
+* `Instance type`: select the type of [worker nodes][docs.using-qovery.configuration.clusters#what-is-a-cluster] you want to deploy to your cluster:
 * `Node auto-scaling`: define the minimum and the maximum number of worker nodes that your cluster can run. The lowest number is the number of worker nodes running on your infrastructure at any time, while the highest number is the maximum number of worker nodes that can automatically be deployed as traffic grows. Please note that a minimum of 3 worker nodes is required to deploy your EKS cluster.
 
+You have the possibility to enable `Karpenter` autoscaler to improve the efficiency and cost of running workloads on your cluster. You can check the [official documentation](https://karpenter.sh/docs/) for more information.
```
Comment (acarranoqovery) [inline website/docs/using-qovery/configuration/clusters.md:217]: only for AWS EKS clusters

Comment (pggb25) [inline website/docs/using-qovery/configuration/clusters.md:217]: maybe add the fact that a SQS queue will created and the associated policy should be added?

File: website/docs/using-qovery/configuration/clusters.md [hunk 4]
```diff
@@ -210,10 +210,23 @@ To confirm, click `Next`.
...
+
+<p align="center">
+  <img src="/img/configuration/clusters/karpenter.png" alt="Enable Karpenter" />
+</p>
+
+Today, only new non-production clusters are supported. It means you won't be able to enable it on your already existing cluster. It will be supported soon.
+
+By activating Karpenter, you have to set:
+* `Disk size`: select the size of the disks to be attached to your cluster instances (to locally store container images etc..).
+* `Default node architecture`: your application will be build using this architecture by default.
```
Comment (acarranoqovery) [inline website/docs/using-qovery/configuration/clusters.md:227]: if you build your applications with the Qovery CI, your application will be builT ...

File: website/docs/using-qovery/configuration/clusters.md (hunk 1 above)
Comment (acarranoqovery) [inline website/docs/using-qovery/configuration/clusters.md:228]: ppot = spot :P

**Review by acarranoqovery — APPROVED** [review]

File: website/docs/using-qovery/configuration/clusters.md (hunk 1 above)
Comment (acarranoqovery) [reply website/docs/using-qovery/configuration/clusters.md:228]: maybe we can add that they should make sure they have the latest IAM permissions from our doc

**Review by acarranoqovery — APPROVED** [review]

**Review by pggb25 — APPROVED** [review]

**Pull request merged**